office-ui-fabric-react/docfx.json

```diff
@@ -70,7 +70,18 @@
       "feedback_system": "None",
       "apiPlatform": "javascript"
     },
-    "fileMetadata": {},
+    "fileMetadata": {
+	  "ms.service": {
```

Comment:
need to use ms.prod.  reasons
1. office is a product instead of service
2. there's no 'office' in ms.service list

Reply:
Updated